agent/metasploit_agent.py

```diff
@@ -103,21 +105,26 @@ def process(self, message: m.Message) -> None:
             if isinstance(results, dict) and results.get("code") == "vulnerable":
                 technical_detail += f'Message: \n```{results["message"]}```'
             else:
-                console_output = self._client.consoles.console(
-                    self._cid
-                ).run_module_with_output(module_instance)
+                console_output = client.consoles.console(cid).run_module_with_output(
+                    module_instance
+                )
                 module_output = console_output.split("WORKSPACE => Ostorlab")[1]
```

Comment:
"WORKSPACE => Ostorlab" should be a const

Reply:
And also check len(console_output.split("WORKSPACE => Ostorlab")) > 1 to avoid crash when accessing [1]